competing-consumers/CompetingConsumers/SenderApp/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace SenderApp
+{
+ using Microsoft.Azure.ServiceBus;
+ using System;
+ using System.Configuration;
+ using System.Text;
+ using System.Threading;
+ using System.Threading.Tasks;
+
+ class Program
+ {
+  // Connection String for the namespace can be obtained from the Azure portal under the 
+  // 'Shared Access policies' section.
+
+  private static readonly string queueName = ConfigurationManager.AppSettings.Get("QueueName");
+  private static readonly string serviceBusConnectionString = ConfigurationManager.AppSettings.Get("ServiceBusConnectionString");
+  private static bool keepRunning = true;
+  private static int msgCounter = 0;
+  static IQueueClient queueClient;
+
+  public static async Task Main(string[] args)
+  {
+
+   AppDomain.CurrentDomain.ProcessExit += new EventHandler(CurrentDomain_ProcessExit);
```

Comment:
Suggestion: Convert this to `HostBuilder` to align with current WebJob expectations.